### DISCUSSION TITLE
GHR Game Recommendations

### DISCUSSION THREAD

**Discussion by qdot** [intiface/intiface-game-haptics-router#35] [General]
Got a game **THAT YOU HAVE TESTED WITH THE GHR AND IT WORKS** that I should add to the [GHR Compatibility Spreadsheet](https://docs.google.com/spreadsheets/d/1g8cZgvumzN3IBDmK2qs48Ms9spJGd4hugJOU4LTMXpk/)? 

Comment here and let me know!

**Comment by AquaGrass** [comment]
Team Fortress 2
Apex Legends

**Comment by erwingame** [comment]
PUBG?

**Comment by qdot** [comment]
Updating this to point out this is for games already tested and needing to be added to the list, not recommendations (I don't think we work with TF2, Apex, or PlUnkBat due to anti-cheat)

**Comment by JessicaKV** [comment]
Did some quick testing for you guys.
Call of Juarez Gunslinger: Vibrates when you get shot and a little bit when you fire several shots in quick succession.

Dishonored 2: From what I tested it vibrates when holding or using the first ability, whenever you get hit, and whenever the heart thing beats. (Which makes some dramatic dialogue absolutely hilarious.

MGSV: Vibrates when firing and reloading most weapons, when diving, during melee tackdowns, while choking someone out or stabbing them, when on a helicopter, when near a helicopter... don't even get me started on cutscenes, I thought lush would break from the intensity of snake petting a dog. because it vibrates with nearly every action. (WARNING: Play online at your own risk)

Metal Gear Rising: Vibrates when you hit most things, almost every time you hit an enemy, and every time you get hit. Vibrates a ton with every hit you land in blademode, somehow making it even better. It also vibrates alot during cutscenes, tldr you're in for a treat

**Comment by VoidField101** [comment]
Vermintide 2: There is probably a mistake in the list, it uses EAC and on the Discord it was mention it doesn't show up on the process List.

Gundam Evolution: Not on the list, but uses EAC therefor won't work. (was also mentioned that it doesn't show up in the list)

**Comment by VantheBat** [comment]
Horizon Zero Dawn seems to be working, just need to select the Steam process rather than the Horizon one.

**Comment by Mintboi88** [comment]
doom eternal works when you jump get hit and shoot etc. works good

**Comment by JillinaV** [comment]
Severed Steel - Works - When you get a kill, when shooting - green - High Speed FPS

Cyberpunk 2077 - Works - If it causes vibrations in game, it works. Explosions, shotguns, riding motorcycle, and getting set on fire do a lot. - Green - FPS RPG

Red Ronin - Works - when you run into a wall, on kill, on death, enemy explosions - Green - Retro Tactical hack n slash

Doom (2016) - Works - Mainly grenades and collectible power ups - Green - Boomer Shooter FPS

Lethal League Blaze - Works - Doesn't do much until you get to crazy speeds. Once that happens, the vibrations are longer and more intense - Green - Brawler

**Comment by cobble1121** [comment]
Would it be possible to support Pavlov VR and bonelab? Thanks either way!

**Reply by qdot** [reply]
I may start looking at VR again with the new version of the system that I'm building. The old Unity support fell apart a long time ago but I'm hoping I can put something more robust together.

**Reply by cobble1121** [reply]
That's awesome, thanks for the update!

**Comment by Cratesmith** [comment]
Rez infinite. Just turn on 'trance vibration' from the controls menu.
Run GHR to connect to controller 2 (you must have a second controller present, Use x360CE to emulate a second controller if you don't have one)

Best to run intiface central on the machine running the game though. Connecting across network (tested via android) seems to make the experience inconsistent.

**Reply by qdot** [reply]
Funny enough, I did a whole video about this one!

https://www.youtube.com/watch?v=rAYdo1vDNak

Just curious: What inconsistency did you get with android?

**Comment by Cratesmith** [comment]
It was devices not acting in time with the game, latency and/or missed events.

It honestly was a bit difficult to be sure which at times.

There were a few warnings in the intiface app, but these were occasional and the vibration response being off or "muddled" was quite consistent

-------- Original Message --------

**Reply by qdot** [reply]
interesting, thanks for letting me know! I'll give it a shot here and see if I can replicate.

**Comment by SeraphinEveles** [comment]
Animal Well works, but only kind of. There's vibration on landing, but it's only transferred over randomly.
